webapi/src/main/scala/org/knora/webapi/messages/admin/responder/listsmessages/ListsMessagesADM.scala

```diff
@@ -848,8 +848,11 @@ case class ListChildNodeADM(
     ListChildNodeADM(
       id = id,
       name = name,
-      labels = labels.sortByStringValue,
-      comments = comments,
+      labels = labels.sortByLanguage,
+      comments = comments match {
+        case None        => None
+        case Some(value) => Some(value.sortByLanguage)
```

Comment:
```suggestion
        case Some(values) => Some(values.sortByLanguage)
```

Reply:
Singular or plural...I made it neutral :)